docs/INSTALL.md

```diff
@@ -156,6 +156,23 @@ sudo dnf install -y \
     sqlite-devel
 ```
 
+_Alpine:_
+
+```bash
+sudo apk add \
+    cmake \
+    freetype-dev \
```

Comment:
What about harfbuzz?

Reply:
`qt6-qtbase-dev` depends on `harfbuzz-dev` so that one is also installed here.